### ISSUE TITLE
Container escape in Docker container

### ISSUE THREAD

**Issue by m-aciek** [Billith/container-escape#4]
Hi, looks like nice piece of work! :) Would it be possible to run all of it inside a container? I'd rather not install and configure nginx locally.

**Comment by linoskoczek** [comment]
You can install it on a virtual machine with Ubuntu Server for example.

**Comment by Billith** [comment]
Yes, that's currently recommended way. However I kinda like idea of running nginx as well as whole python application in separate containers. It would require pretty big changes, but I think it's doable. I will give it a try in the near future, currently I don't want to make such a significant changes to the project for few reasons. Thank you for that suggestion @m-aciek.
